diff --git a/node_modules/@wokcito/ffmpeg-kit-react-native/android/build.gradle b/node_modules/@wokcito/ffmpeg-kit-react-native/android/build.gradle
index 2456c18..6751799 100644
--- a/node_modules/@wokcito/ffmpeg-kit-react-native/android/build.gradle
+++ b/node_modules/@wokcito/ffmpeg-kit-react-native/android/build.gradle
@@ -30,11 +30,11 @@ android {
     namespace 'com.arthenica.ffmpegkit.reactnative'
   }
 
-  compileSdkVersion 33
+  compileSdkVersion rootProject.ext.has('compileSdkVersion') ? rootProject.ext.get('compileSdkVersion') : 35
 
   defaultConfig {
     minSdkVersion safeExtGet('ffmpegKitPackage', 'https').contains("-lts") ? 16 : 24
-    targetSdkVersion 33
+    targetSdkVersion rootProject.ext.has('targetSdkVersion') ? rootProject.ext.get('targetSdkVersion') : 35
     versionCode 602
     versionName "6.0.2"
   }
