diff --git a/node_modules/ffmpeg-expo/android/build.gradle.kts b/node_modules/ffmpeg-expo/android/build.gradle.kts
index 54da5e9..ee872b2 100644
--- a/node_modules/ffmpeg-expo/android/build.gradle.kts
+++ b/node_modules/ffmpeg-expo/android/build.gradle.kts
@@ -62,5 +62,5 @@ dependencies {
     implementation("org.jetbrains.kotlinx:kotlinx-coroutines-android:1.7.3")
     
     // Expo modules
-    implementation("expo.modules:expo-modules-core:1.11.8")
+    implementation(project(":expo-modules-core"))
 }
